lib/rules/no-forbidden-elements.js

```diff
@@ -32,6 +32,16 @@ function sanitizeConfigArray(forbidden = []) {
   return forbidden.filter(Boolean).sort((a, b) => b.length - a.length);
 }
 
+function hasHeadParent(path) {
+  let parents = [...path.parents()];
+  let headElementInParentPath = parents.find(
+    (parent) => parent.node.type === 'ElementNode' && parent.node.tag === 'head'
+  );
+  if (headElementInParentPath) {
+    return true;
+  }
+  return false;
```

Comment:
This could be simplified a bit:

```suggestion
  return Boolean(headElementInParentPath);
```